misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/ChronoLocalDateAssertionSpec.kt

```diff
@@ -89,6 +93,21 @@ abstract class ChronoLocalDateAssertionSpec(
                     }.toThrow<AssertionError> { messageContains("$isAfterDescr: $december24") }
                 }
             }
+            describe("${isBeforeOrEquals.name} ...") {
```

Comment:
```suggestion
            describe("${isAfterOrEquals.name} ...") {
```
But I'll fix it myself